components/instalation/instalation.css

```diff
@@ -0,0 +1,53 @@
+@import url(/landing.css);
+@import url(/variable.css);
+
+body{
+    display: flex;
+    max-width: 100%;
+    flex-wrap: wrap;
+}
+*{
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+
+.drawer_component{
+    display: flex;
+    /* border: 1px solid black; */
+    /* position: relative; */
+
+    
```

Comment:
You could avoid commented code and empty lines.

Reply:
okk i will correct it.